core/drivers/crypto/se050/ext/sub.mk

```diff
@@ -0,0 +1,42 @@
+include core/drivers/crypto/se050/cflags.mk
```

Comment:
Shouldn't this whole file be part of the upstream library code? (if a file is renamed for example, this would need updating).

Reply:
yes, will fix that.